aries-site/src/examples/templates/ScreenContainer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, ResponsiveContext } from 'grommet';
+import { SidebarExample } from '.';
+
+export const ScreenContainer = ({ mobile, ...rest }) => (
+  <ResponsiveContext.Provider value={mobile && 'small'}>
+    <ResponsiveContext.Consumer>
+      {size => (
+        <Box
+          background="background-back"
+          width={size === 'small' ? { max: 'large' } : '100%'}
+          height={{ max: 'large' }}
+          style={{ position: 'relative' }}
+        >
+          <Box direction="row" fill>
+            {size !== 'small' && <SidebarExample />}
+            <Box
+              overflow="auto"
+              pad={{ horizontal: 'medium', bottom: 'medium' }}
+              flex
+              {...rest}
+            />
+          </Box>
+          {size === 'small' && (
+            <SidebarExample
+              direction="row"
+              fill="horizontal"
+              mobile={mobile}
```

Comment:
Are we keeping the `mobile` calculation way? what about another screen sizing?

Reply:
Good catch, we don't need it anymore. I'll remove this. We are using `size` and `useContext` in Sidebar to achieve this.